api/catalog/api/views/media_views.py

```diff
@@ -138,7 +137,7 @@ def report(self, request, *_, **__):
         identifier = media.identifier
         serializer = self.get_serializer(data=request.data)
         if not serializer.is_valid():
-            raise get_api_exception("Invalid input.", 400)
+            raise ValidationError("Invalid input.")
```

Comment:
This could be replaced with the serializer's own error and we'd get something more descriptive and helpful in the end:

```suggestion
        serializer.is_valid(raise_exception=True)
```

The DRF docs about this pattern are here: https://www.django-rest-framework.org/api-guide/exceptions/#validationerror